bylaws.html

```diff
@@ -252,80 +250,74 @@ <h3 id="voting">3.3. Voting</h3>
 
-<p>All formal voting must be done in an email thread with the appropriate <a href="#tags">subject tag</a>. Formal votes may contain multiple items for approval and these should be clearly separated. Formal voting is then carried out by replying to the vote mail. Formal votes are open for a period of at least 72 hours to allow all active voters time to consider the vote. Votes can be held open longer than this at the discretion of the person who initiated the vote.
+<p>You must conduct all formal voting in an email thread with the appropriate <a href="#tags">subject tag</a>. Formal votes may contain multiple items for approval, and these should be separated clearly. Formal voting is then carried out by replying to the vote mail. Formal votes are open for a period of at least 72 hours to allow all active voters time to consider the vote. Votes can be held open longer than this at the discretion of the person who initiated the vote.
 
-<p>Votes on <a href="#types">PMC decisions</a> are binding if they are cast by a PMC member. For all other purposes, votes are binding if they are cast by a committer. However, it is important to remember that all participants on a list get a vote. And you are encouraged to vote, even if your vote is not binding. This is a good way to get involved in the project and helps to inform the decision-making process.
+<p>Votes on <a href="#types">PMC decisions</a> are binding if a PMC member casts them. For all other purposes, votes are binding if a committer casts them. However, it is essential to remember that all participants on a list get a vote. Moreover, you are encouraged to vote, even if your vote is not binding. Voting is an excellent way to get involved in the project and helps to inform the decision-making process.
 
-<p>You are encouraged to use an informal voting model to take a quick poll or to wrap up a discussion, whether you are a committer yet or not. These votes are informal and can be initiated by anyone. Binding votes are only needed for project-level decision-making.
+<p>You are encouraged to use an informal voting model to take a quick poll or to wrap up a discussion, whether you are a committer yet or not. Anyone can initiate these votes.
 
-<h3 id="approval">3.4. Approval Models</h3>
+<h3 id="approval">3.4. Approval models</h3>
 
 <p><strong>We use three different approval models for formal voting</strong>:
 
   <ul>
     <li>RTC (see section 3.5)
-      <ul>
-	<li>Requires one binding +1 vote and no vetos
-      </ul>
+      <ul><li>Requires one binding +1 vote and no vetos</ul>
```

Comment:
I think if you do this, you want to add the `</li>` before the `</ul>` for best results. Might as well fix this in the entire list too.

-0.1 on collapsing it to a single line, but *shrug*